op-node/benchmarks/batchbuilding_test.go

```diff
@@ -0,0 +1,128 @@
+package benchmarks
+
+import (
+	"fmt"
+	"math/big"
+	"math/rand"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-batcher/compressor"
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-service/testutils"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+func RandomSingularBatch(rng *rand.Rand, txCount int, chainID *big.Int) *derive.SingularBatch {
```

Comment:
This function was taken from the derive package's tests. Even though it seemed to be exported I couldn't access it, and didn't think harder about it. I should double check if I was just doing something wrong or if I really need to copy this function.

Reply:
It could be that the package name for the tests is different

Reply:
Same package name for the test.

Looked this up real quick, https://www.reddit.com/r/golang/comments/jjmsoc/how_to_share_test_utilities_accros_packages/

Looks like Exported functions that are in `_test` files still don't export across packages, not something I've dealt with before. I moved it into a `batcher_test_utils.go` with the same package name and all, that did the trick.